Docs/source/install/hpc/spock.rst

```diff
@@ -35,7 +35,7 @@ We use the following modules and environments on the system (``$HOME/warpx_spock
    export proj=<yourProject>
 
    # required dependencies
-   module load cmake
+   module load cmake/3.20.0
```

Comment:
Just got a report from another AMReX project related to https://gitlab.kitware.com/cmake/cmake/-/issues/21887

We should use `cmake/3.20.2` on Spock to have the patch for this, too - even if it does not affect it yet. (Fixes landed in 3.20.1+)